src/lib.rs

```diff
@@ -291,7 +291,7 @@ mod tests {
             "vacuum",
         ];
         for pass in pass_list.iter() {
```

Comment:
This only validated that no pass was removed. Was there any new pass(es) introduced in the new binaryen version? If so, that should be listed here.

Reply:
Good point! I think there should be actually